### ISSUE TITLE
./GunetherRuhe/RuheNayebi.md

### ISSUE THREAD

**Issue by timm** [ds4se/chapters#43]
After review, relabel to 'reviewTwo'. After second review, relabel to 'EditorsComment'.

**Comment by hongyujohn** [comment]
## Title of chapter

What Counts is Decisions, Not Numbers – Towards an Analytics Design Sheet

## URL to the chapter

https://github.com/ds4se/chapters/blob/master/GunetherRuhe/RuheNayebi.md

## Message?

This chapter describes a high-level decision-making process for software products. The authors look at this process from the analytics perspective and define what can be provided at each step.

## Accessible?

Yes.

## Size?

Yes.

## Gotta Mantra?

Section 3 "The Analytics Design Sheet" is not very clear to me. It is not clear how each quadrant (Q1 to Q4) is created, and the relationship among them. For example: it says that "Q3 evaluates data features and provides the status quo in terms of quantity and quality of the data for conducting analytics". You may make it explicit how it is done (automatically or manually?).

For the illustrative example, how the results of Q4 are obtained could be described more. For example, for the point about "Clustering of stakeholders based on the commonality of their preferences against stated objectives", how is it derived from Q3/Q2? 

## Best Points

An interesting idea of making SE decisions based on analytics.

## Minor point

A few typos:
select-ed => selected
sup-port => support 

There is a relate work, which proposes the selection of design alternatives for architectural design:
Hongyu Zhang and Stan Jarzabek, A Bayesian Network Approach to Rational Architectural Design, International Journal of Software Engineering and Knowledge Engineering, vol. 15 (4), World Scientific, August 2005, pp. 695-717.

**Comment by timm** [comment]
need a better  reference for [6] 
- could not find a definitive document defending "ust about 20% of the features implemented in a software product are often or always used, while 64% of the features get rarely or never used [6]. This means that substantial development effort is wasted. But even worse: it means that maintenance cost of the product (constituting the majority of the total product effort) is higher than needed. This often implies delays in delivering products or lack of quality"

i had a look around and maybe these will do better than [6]:
- https://blogs.msdn.microsoft.com/b8/2011/08/29/improvements-in-windows-explorer/
- https://github.com/txt/se16/blob/master/todo/howDevelopersUseEclipseIDE.pdf

**Comment by timm** [comment]
This proposal sounds.. familiar. E.g. 
- balanced score cards http://balancedscorecard.org/Resources/About-the-Balanced-Scorecard), 
- house of quality, 
- design rationale 
  - see chapters in http://www.amazon.com/Design-Rationale-Techniques-Computers-Cognition/dp/080581566X
  - http://rationale.csail.mit.edu/; 
  - see list of systems in http://web.cs.wpi.edu/Research/aidg/DR-Rpt98.html

So maybe a little more on related work?

Also, the actual connection to data science is a little tenuous.  The idea is good (know the goals before collecting the wrong data) but can you expand on that a little.

**Comment by timm** [comment]
Also, it would be good to see some full example of using these higher goals to drive data collection and interpretration (e.g. is this a theoretical framework or a case study)?

**Comment by timm** [comment]
mentioned in #124 #23

**Comment by maleknaz** [comment]
Thanks everyone for the very good feedback.
@timm 
- We dropped [6] and replaced the motivation section with better description.
- Yes, we agree that it might be somehow related to QFD. However, the idea is more related to GQM and GQM+.
- We changed the context of the example and described it in more detail to clear the pointed ambiguities. 

@hongyujohn 
- We changed the description and provided more a tangible example for the four quartiles. The sheet is aimed to be used as a guideline for human decision makers.
-  We dropped that example and replaced it with another one to make the navigation between quartiles more clear.
- Typos fixed.
